origin.spec

```diff
@@ -101,7 +101,13 @@ Requires:       docker-io >= %{docker_version}
 Requires:       tuned-profiles-%{name}-node = %{version}-%{release}
 Requires:       util-linux
 Requires:       socat
+# storage plugins
+Requires:       e2fsprogs
+Requires:       xfsprogs
 Requires:       nfs-utils
+Requires:       ceph-common
+Requires:       glusterfs-fuse
+Requires:       iscsi-initiator-utils
```

Comment:
Well poop. Do you even need the Dockerfile change with this change? Do we really want to force all of these on the host...   ugggh, no good answers.

I think I would do the minimal docker file change and not the .spec change, but that's just me.